### ISSUE TITLE
Not using identifier

### ISSUE THREAD

**Issue by neel** [ktquez/vue-disqus#42]
This is what I am using

    <Disqus shortname="http-website-org" :pageConfig="disqus"></Disqus>

where `disqus` is comming from `data`

```
new Vue({
    router,
    el: '#main',
    data(){
        return {
            // ...
            disqus: {
                url: 'https://website.org/spatial/'+synopsis.id,
                identifier: '/spatial/'+synopsis.id
            }
        };
    },
  // ...
}
```
But that didn't work. In all pages I see the same comments. So I added the `identifier` inside the html directly.

    <Disqus shortname="http-website-org" :pageConfig="{identifier: '/spatial/'+synopsis.id, url: 'https://website.org/spatial/'+synopsis.id}"></Disqus>

That also didn't work.

I was not using `vue-router`. But `Disqus` calls to `this.$router`. So I added an empty router.

```
Vue.use(VueRouter);

const routes = [];
const router = new VueRouter({
  routes // short for `routes: routes`
})

Vue.use(VueDisqus, {
    shortname: 'http-website-org'
})

new Vue({
    router,
    el: '#main',
   // ...
}

```

**Comment by ktquez** [comment]
Hi @neel, thanks for using.

I went up a correction on line [136](https://github.com/ktquez/vue-disqus/blob/master/src/Disqus.vue#L136)

Could you test with [v4.0.1](https://github.com/ktquez/vue-disqus/tree/v4.0.1)?

**Comment by wjfergus** [comment]
I'm having the same issue. Where the identifier is not being recognized and thus showing all comments. I tested with the most recent version which is 4.0.1.

**Comment by ktquez** [comment]
Could you do an example online and the code of how vue-dials are being used?

**Comment by wjfergus** [comment]
Totally will. This is my first time posting to a github project I'm
considering using. Hopefully I can provide value back to your work (re:
hopefully it's not a use error!)

Billy

**Comment by wjfergus** [comment]
Using vue-disqus@4.0.1

main.js:
```
import VueDisqus from 'vue-disqus'

Vue.use(VueDisqus)
```
Using Disqus in a component:
```
<template>
  <Disqus shortname="example" :pageConfig="pageConfig"/>
</template>

<script>
   import { Disqus } from 'vue-disqus'    

    export default {
        name: 'exampleBlogComments',
        components: { Disqus },
        props: ['content'],
        data: function () {
            return {
                pageConfig: {
                    identifier: 'test',
                }
            }
        }
    }
</script>
```

No matter what I change the variable identifier, it always loads in the same comment threads. Hopefully somebody can help figure out how I configured this wrong?

**Comment by pang-lee** [comment]
So am I same issue, and I ran out another problem is that I want to use pop out window to show the disqus, but every post run out same disqus in per page, and pageConfig didn't work will

**Comment by ktquez** [comment]
@wjfergus 
Are you using vue-router? 
If yes, are you using history mode?

**Comment by Radpc** [comment]
I'm using 4.0.1, with vue-router and history mode active, and can confirm that it wasn't functioning properly, same kind of error.

I think I didn't setup right at first, and created some _comments_. It may have bugged something, but when I deleted the comments from the site, setup properly and then tried again, everything worked out just fine.

I also setted the title to be different for every comment page, so maybe there's that too.
